### DISCUSSION TITLE
How To Upload Specific Apk?

### DISCUSSION THREAD

**Discussion by inyomanw** [Triple-T/gradle-play-publisher#1030] [Q&A]
Hi! May I know if this library can do uploading a specific apk like App Distribution Build Parameters (apkPath/artifactPath) in appDistribution library?

I have 2 APK on my productFlavors and i want to upload app-production-release-b.apk instead of app-production-release.apk

I have tried using ```artifactDir.set(file("app/build/outputs/apk/production/release/app-production-release-b.apk"))```  like this, but it didn't work

**Comment by SUPERCILEX** [comment]
Unfortunately, there's not enough detail to know how to help. I'm guessing you're looking for something like this: https://github.com/Triple-T/gradle-play-publisher#disabling-publishing. You can selectively enable publishing for certain variants.
